<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby</class>
 <widget class="QWidget" name="Lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лобби</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>image/window.png</normaloff>image/window.png</iconset>
  </property>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Start_game">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Готов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Refresh_lobby">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lobby_name">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ЛОББИ</string>
   </property>
  </widget>
  <widget class="QLabel" name="Nickname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>671</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="Enter_lobby">
        <property name="text">
         <string>Войти в лобби</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Create_lobby">
        <property name="text">
         <string>Создать лобби</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Exit_lobby">
        <property name="text">
         <string>Выйти из лобби</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Delete_lobby">
        <property name="text">
         <string>Удалить лобби</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Stat_game">
        <property name="text">
         <string>Статистика</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
